agent/engine/docker_task_engine.go

```diff
@@ -1606,6 +1607,8 @@ func (engine *DockerTaskEngine) pullAndUpdateContainerReference(task *apitask.Ta
 
 	findCachedImage := false
 	if !pullSucceeded {
+		// Extend error message
+		metadata.Error = apierrormsgs.AugmentNamedErrMsg(metadata.Error)
```

Comment:
it seems like this is the only place where `AugmentNamedErrMsg` is used? Is there some plan to add more errors to this?

Reply:
At this time, there are no plans to continue updating other messages using `AugmentNamedErrMsg`. This function was implemented and applied to specifically for the scenarios we are trying to address. If future requirements necessitate similar enhancements for additional error messages, we can consider extending its usage then.